app/src/main/java/eu/kanade/presentation/more/onboarding/PermissionStep.kt

```diff
@@ -117,28 +118,30 @@ internal class PermissionStep : OnboardingStep {
                 },
             )
 
-            HorizontalDivider(
-                modifier = Modifier.padding(vertical = 8.dp, horizontal = 16.dp),
-                color = MaterialTheme.colorScheme.onPrimaryContainer,
-            )
+            if (!isFossFlavor) {
```

Comment:
```suggestion
            if (isFossFlavor) return@Column 
```
and revert the other bit